addons/hr/static/src/components/avatar_card_employee/avatar_card_employee_popover.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="hr.AvatarCardEmployeePopover" t-inherit="mail.AvatarCardPopover">
+        <xpath expr="//span[hasclass('o_avatar')]/img" position="attributes">
+            <attribute name="t-attf-src">/web/image/{{props.employeeModel}}/{{props.id}}/avatar_128</attribute>
+        </xpath>
+        <xpath expr="//span[hasclass('o_card_avatar_im_status')]" position="replace">
+            <span t-if="employee.user_id" name="icon" class="o_card_avatar_im_status position-absolute d-flex align-items-center justify-content-center o_user_im_status">
+                <i t-if="employee.im_status === 'online'" class="fa fa-fw fa-circle text-success" title="Online" role="img" aria-label="User is online"/>
+                <i t-elif="employee.im_status === 'away'" class="fa fa-fw fa-circle text-warning" title="Idle" role="img" aria-label="User is idle"/>
+                <i t-elif="employee.im_status === 'offline'" class="fa fa-fw fa-circle-o text-700" title="Offline" role="img" aria-label="User is offline"/>
+                <i t-elif="employee.im_status === 'bot'" class="fa fa-fw fa-heart text-success" title="Bot" role="img" aria-label="User is a bot"/>
+                <i t-elif="employee.im_status" class="fa fa-fw fa-question-circle" title="No IM status available"/>
+            </span>
+            <span t-elif="employee.show_hr_icon_display" name="icon" class="o_card_avatar_im_status position-absolute d-flex align-items-center justify-content-center o_employee_presence_status">
+                <!-- Employee is present/connected and it is normal according to his work schedule  -->
+                <i t-if="employee.hr_icon_display == 'presence_present'" class="fa fa-fw fa-circle text-success" title="Present" role="img" aria-label="Employee is present"/>
+                <!-- Employee is not present/connected and it is normal according to his work schedule -->
+                <i t-if="employee.hr_icon_display == 'presence_absent'" class="fa fa-fw fa-circle-o text-warning" title="Absent" role="img" aria-label="Employee is absent"/>
+                <!-- Employee is connected but according to his work schedule, he should not work for now  -->
+                <i t-if="employee.hr_icon_display == 'presence_absent_active'" class="fa fa-fw fa-circle-o text-success" title="Active but not present" role="img" aria-label="Employee is active but not present"/>
+                <!-- Employee is not here but according to his work schedule, he should be connected -->
+                <i t-if="employee.hr_icon_display == 'presence_to_define'" class="fa fa-fw fa-circle-o text-muted" title="No information" role="img" aria-label="No information"/>
```

Comment:
> I am not sure if "No information" is the correct description if it should describe the comment above. But I don't know how to differentiate it from "Absent" so yeah, 🤷

I tried to find clear terms to define those status but indeed not really convincing. I will stick to the one used in hr ("To define") and let them modify that if they think it is needed. At least it will be consistent between the views in hr.

Reply:
Yeah sure, we don't have to re-invent the wheel, we can keep them as is